lib/helpers/rule-helper.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @fileoverview Base for all rules
```

Comment:
Please update with an accurate description. The word "base" typically applies to inheritance and thus implies that rules extend this file, which they do not.